Assets/Scripts/Tank/PlayerTank.cs

```diff
@@ -0,0 +1,98 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerTank : GenericSingleton<PlayerTank>
+{
+    public int m_PlayerNumber = 1;
```

Comment:
If we are not using things outside the class we should make these things private .